configuration/configuration.go

```diff
@@ -145,6 +145,18 @@ type Configuration struct {
 	Health Health `yaml:"health,omitempty"`
 
 	Proxy Proxy `yaml:"proxy,omitempty"`
+
+	// Schema1Compability configures how schema1 manifests will be handled
+	Schema1Compatibility struct {
+		// TrustKey is the signing key to use for adding the signature to
+		// schema1 manifests.
+		TrustKey string `yaml:"signingkey,omitempty"`
```

Comment:
I wonder if this should be called `signingkeyfile` to make it more obvious that this is a path rather than an inline key. It's probably obvious to most people, though.